examples/galaxies/plot_size.py

```diff
@@ -0,0 +1,136 @@
+"""
+Galaxy size
+===========
+
+This example demonstrate how to obtain sizes for early and late type galaxies
+in SkyPy.
+
+"""
+
+
+# %%
+# Size-magnitude relation
+# -------------------------
+#
+# According to Shen et al. 2003 [1]_, the observed size of galaxies
+# and the absolute magnitudes follow simple analytic formulae. For early-type
+# galaxies, the relation (14) reads
+#
+# .. math::
+#
+#    log (\bar{R}/kpc) = -0.4aM + b,
+#
+# with :math:`a` and :math:`b` fitting constants. Likewise, for late-type galaxies
+# the formula (15) reads
+#
+# .. math::
+#
+#    log (\bar{R}/kpc) = -0.4\alpha M +(\beta -\alpha) log \left[1 + 10^{-0.4(M-M_0)}\right] +\gamma
+#
+# with a dispersion given by equation 16
+#
+# .. math::
+#
+#    \sigma_{ln R} = \sigma_2 + \frac{\sigma_1 - \sigma_2}{1 + 10^{-0.8(M - M_0)}}
+#
+# where :math:`\alpha`, :math:`\beta`, :math:`\gamma`, :math:`\sigma_1`, :math:`\sigma_2` and
+# :math:`M_0` are fitting parameters.
+#
+# In SkyPy, we draw physical sizes of both galaxy types from a lognormal distribution
+#
+# .. math::
+#
+#    p(R) = \frac{1}{\sigma R \sqrt{2 \pi}}e^{- \frac{(ln R - \mu)^2}{2 \sigma^2}}
+#
+# with mean :math:`\mu = 0` and standard deviation :math:`\sigma`, and an amplitude
+# :math:`\bar{R}` given by the Shen et al. 2003 model:
+# :func:`skypy.galaxy.size.early_type_lognormal()` and
+# :func:`skypy.galaxy.size.late_type_lognormal()`
+# (c.f. equations above).
+#
```

Comment:
Not necessary

Reply:
What is not necessary, the equation or the explanation?

Reply:
neither is I think, that it is a lognormal distribution is literally the model